src/EventSourcingTests/EventSourcingTests.csproj

```diff
@@ -156,9 +156,9 @@
     </ItemGroup>
 
     <ItemGroup>
-      <Reference Include="Microsoft.Extensions.Hosting">
-        <HintPath>..\..\..\..\..\..\usr\local\share\dotnet\shared\Microsoft.AspNetCore.App\6.0.21\Microsoft.Extensions.Hosting.dll</HintPath>
-      </Reference>
+        <Reference Include="Microsoft.Extensions.Hosting">
+            <HintPath>..\..\..\..\..\..\usr\local\share\dotnet\shared\Microsoft.AspNetCore.App\6.0.21\Microsoft.Extensions.Hosting.dll</HintPath>
```

Comment:
Why is this `<HintPath>` needed?

Reply:
Good question, but I don't know 🤷‍♂️ @jeremydmiller added it in https://github.com/JasperFx/marten/pull/2722. Of what I see it either looks redundant or like some DI voodoo. Maybe @jeremydmiller can expand on that.